modules/contrib/smartstatus.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, config, theme):
         self.devices = self.list_devices()
         self.display = self.parameter("display", "combined")
         self.drives = self.parameter("drives", "sda")
+        self.show_names = self.parameter("show_names", "True")
```

Comment:
For this, you can use `util.format.asbool()` to unify boolean handling - that automatically allows a user to use yes/no, true/false, 0/1 for the parameter.

Reply:
Man, I was wondering why the True False boolean was not working in your code!